doc/jointcal.tex

```diff
@@ -29,527 +43,603 @@
-residual ntuples and hope that the first serious users will contribute
-plotting tools.
+optimal in a statistical sense.
+
+This package performs a least-square fit to a set of images. Since
+it aims at statistical optimality, we maximize the likelihood of the
+measurements with respect to all unknown parameters required to describe
+the data. These parameters consists mostly in two sets: the position
+(on the sky) of the objects in common, and the mapping of each image
+to the sky. To these obvious parameters, one can add proper motions
+(where applicable), and parameters describing the differential effect
+of atmospheric refraction on the position of objects. It is clear
+that one cannot fit simultaneously the position on the sky and the
+mappings from CCD coordinates to the sky, without extra constraints:
+the ``sky'' coordinate system is then undefined, and one needs reference
+positions in order to fully define this frame. So far, we have used
+the USNO (A 2.0) catalog for this purpose.
+
+SCAMP\footnote{see \href{http://www.astromatic.net/software/scamp}{http://www.astromatic.net/software/scamp}}
+is the reference package for simultaneous astrometry in astronomy,
+at least for relative alignment of wide-field images prior to stacking.
+Regarding optimization, SCAMP follows a somehow different route from
+ours: it does not optimize over the position of common objects but
+rather minimizes the distance between pairs of transformed measurements
+of the same object. This approach is not a maximum likelihood optimization,
+and hence is likely sub-optimal statistics wise. We do not know how
+sub-optimal it is, but the main drawback of SCAMP in the context of
+LSST is the fact that it is a program and not a library, and hence
+not flexible regarding formats of images and catalogs. But since SCAMP
+has been used for almost a decade in production by various teams,
+the quality checking tools it provides should likely be reproduced
+in the context of our package. We provide residual ntuples and hope
+that the first serious users will contribute plotting tools.
 
 The code is heavily biased towards astrometry. One related problem,
-important in some of the target applications, is relative {\it
-  photometry}.  We have not provided yet the determination of relative
-flux scales between images but we plan to do so. The principles are
-similar to relative astrometry, but considerably simpler. The main reason
-to integrate it to the present package is that loading the input catalogs
+important in some of the target applications, is relative \textit{photometry}.
+We have not provided yet the determination of relative flux scales
+between images but we plan to do so. The principles are similar to
+relative astrometry, but considerably simpler. The main reason to
+integrate it to the present package is that loading the input catalogs
 takes a large amount of time and so, solving for the relative photometry
 once they are in memory looks like a good idea.
```

Comment:
"Looks like a good idea" -> "is likely to be more efficient" (or similar)

Reply:
I replaced that entire paragraph, since I now have a photometric model implemented.